torch/csrc/jit/codegen/cuda/fusion_segmenter.cpp

```diff
@@ -1930,6 +1930,7 @@ void TranslateApplicableWelford::translateSingleWelford(WelfordOp* welford) {
   //  largely taken from batchnorm cpp benchmark
   auto& in_root = in_val->getRootDomain();
   auto& out_root = out_avg->getRootDomain();
+  TORCH_INTERNAL_ASSERT(in_root.size() == out_root.size());
```

Comment:
Naive comment, looking at the logic below, the check here should be `in_root.size() <= out_root.size()`?
```
for (const auto i : c10::irange(in_root.size())
   ... out_root[i] ...
```

Reply:
sure, good point - I'll change that.

@jjsjann123 is this PR still useful now that the actual fixes are landed?

Reply:
Yes, let's merge this one.